packages/protocol/contracts/stability/Reserve.sol

```diff
@@ -306,6 +310,16 @@ contract Reserve is
     emit SpenderRemoved(spender);
   }
 
+  function addExchangeSpender(address spender) external onlyOwner {
```

Comment:
What do you think about adding a view call that exposes `isExchangeSpender`? I also wonder if it would be useful to have an array of all exchange spenders since that would be harder to add later?

Reply:
pretty sure `isExchangeSpender` is generated automatically

Reply:
On the web3 contract, or for other smart contracts?

Reply:
I would probably explicitly add this to the interface of `IReserve`

Reply:
Why `addSpender` and `removeSpender` are not in the IReserve?

Reply:
I added `addExchangeSpender` and `removeExchangeSpender ` to the `IReserve` file.

Reply:
I was thinking of `isExchangeSpender` so that other contracts can access, but not super necessary, but might be nice to create a ticket for it then